### PR TITLE
Feature/evo 22

### PR DESCRIPTION
**Pull request by mrhodas107**
Payment information can now be viewed in OrderDeatilsPage.

### CODE REVIEW

File: app/containers/OrderPage/OrderDetailPage/PaymentInformation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+// remove, use babel for antD css
+import { Col, Card, Icon, Button, List } from 'antd';
+import 'antd/dist/antd.css';
+
+import * as React from 'react';
+
+import { paymentDetailData } from './constants';
+
+export default function OrderDetailHeader() {
```
Comment (jmiller-cvet) [inline app/containers/OrderPage/OrderDetailPage/PaymentInformation/index.tsx]: Wrong name here as well.

File: app/containers/OrderPage/OrderDetailPage/ShippingInformation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+// remove, use babel for antD css
+import { Col, Card, Icon, Button, List } from 'antd';
+import 'antd/dist/antd.css';
+
+import * as React from 'react';
+
+import { shippingDetailData } from './constants';
+
+export default function OrderDetailHeader() {
```
Comment (jmiller-cvet) [inline app/containers/OrderPage/OrderDetailPage/ShippingInformation/index.tsx]: Wrong name for the function.

**Comment by mrhodas107** [comment]
All these are now fixed in EVO-23.

**Comment by jmiller-cvet** [comment]
I'll wait to merge EVO-23 until you either abandon EVO-21 and EVO-22 or update them so I can merge them first.

**Pull request closed without merging**